libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -436,6 +437,24 @@ std::shared_ptr<WaveTrack> WaveTrackFactory::Create(sampleFormat format, double
    return std::make_shared<WaveTrack>(mpFactory, format, rate);
 }
 
+TrackListHolder WaveTrackFactory::Create(size_t nChannels)
+{
+   return Create(nChannels, QualitySettings::SampleFormatChoice(), mRate.GetRate());
+}
+
+TrackListHolder WaveTrackFactory::Create(size_t nChannels, sampleFormat format, double rate)
+{
+   auto channels = std::vector<std::shared_ptr<Track>>{ };
+   std::generate_n(
```

Comment:
Thank you for making me look up this standard algorithm and refresh my memory of it.